packages/cozy-mespapiers-lib/src/helpers/queries.js

```diff
@@ -18,7 +18,7 @@ export const buildFilesQueryByLabels = labels => {
           trashed: false
         })
         .indexFields(['metadata.qualification.label'])
-        .select(['metadata.qualification.label'])
+        .select(['name', 'metadata.qualification.label'])
```

Comment:
`Thus, when an index is required, any field involved in a selector should be indexed.`
https://docs.cozy.io/en/tutorials/data/queries/#index-fields

Reply:
Involved in a **selector**, not a `select` ;)

Reply:
bah on s'est dit l'inverse en vocal @paultranvan , on fait confiance à la doc ou on modifie la doc ?

Reply:
Réveil difficile, my bad. Je sais pas pourquoi je voyais un where à la place du select :'(

Reply:
@JF-Cozy In my previous comment, you should trust :smile: